setup.py

```diff
@@ -13,12 +13,12 @@
     "attrs>=18.2.0,<20.0.0",
     "colored>=1.3.92,<2.0.0",
     "typing_extensions>=3.6,<4.0.0",
+    "pytest>=5.0.0,<6.0.0",
 ]
 test_requires = [
     "codecov",
     "coverage[toml]",
     "invoke",
-    "pytest",
```

Comment:
nb. should it be in both? As in both `install_requires` and `test_requires`

Reply:
we always install "install requires", so I don't think it's necessary